src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,41 @@
+package org.example;
+
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class Main
+{
+    private static final Logger log = Logger.getLogger(Main.class);
+    private static final Logger orderLogger = Logger.getLogger("SECOND_LOGGER");
+
+    public static void main( String[] args )
```

Comment:
Let's split your main method into several (eg. method to get info, method to validate price, method to write file) and test some of them - it will be easier